tests/test_stt.py

```diff
@@ -25,15 +25,15 @@ def read_wav_file(filename: str) -> rtc.AudioFrame:
 
 
 async def test_recognize():
-    stts = [deepgram.STT(), google.STT(), openai.STT()]
+    # stts = [deepgram.STT(), google.STT(), openai.STT()]
```

Comment:
uncomment test code

Reply:
yep, going to do but google and openai aren't changed yet